lib/commands/find.js

```diff
@@ -54,7 +54,14 @@ helpers.findNativeElementOrElements = async function (strategy, selector, mult,
   }
 
   if (strategy === 'xpath') {
-    selector = selector.replace(/(UIA)([^\[\/]+)/g, (str, g1, g2) => {
+    // Replace UIA if it comes after a forward slash
+    selector = selector.replace(/\/(UIA)([^\[\/]+)/g, (str, g1, g2) => {
+      rewroteSelector = true;
+      return '/' + stripViewFromSelector(`XCUIElementType${g2}`);
+    });
+
+    // Replace UIA if it is at the beginning of the selector string
+    selector = selector.replace(/^(UIA)([^\[\/]+)/, (str, g1, g2) => {
```

Comment:
Can these two regexes be reduced to a single expression? A regex that produces similar results (replacing "UIA" both after a forward slash and beginning of a string) can be:
```
selector = selector.replace(/(^|\/)(UIA)([^\[\/]+)/, (str, g1, g2, g3) => {
  rewroteSelector = true;
  return '/' + stripViewFromSelector(`XCUIElementType${g2}`);
});
```